src/config/loader.ts

```
@@ -1,78 +1,545 @@
+    catch (error) { console.error("[kokpit] could not remove installed settings temporary file:", error); }
+    try { unlinkSync(CONFIG_DISPLACED_PATH); }
+    catch (error) { console.error("[kokpit] could not remove completed settings transaction:", error); }
+    return true;
+  } finally {
+    if (!installed && existsSync(CONFIG_DISPLACED_PATH)) {
+      try {
+        linkSync(CONFIG_DISPLACED_PATH, CONFIG_PATH);
+        try { unlinkSync(CONFIG_DISPLACED_PATH); }
+        catch (error) { console.error("[kokpit] could not remove restored settings transaction:", error); }
+      } catch (error) {
+        if (!hasErrorCode(error, "EEXIST")) throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)